requirements.txt

```diff
@@ -1,5 +1,4 @@
-ansible
-ansible-base
```

Comment:
Do we need to keep `ansible-base`?

Reply:
I presume you've tested building with this and have successfully built an image as a test? I think it should be good to go if so.

Reply:
The image-builder builds and runs fine with this, resolving.